test/tags/_Mail.cfc

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
this is the old tests, just disabled?

Reply:
I think these tests came from the 5.3.9 branch, but there is unit tests coverage for the changes. You can see there are tests for the 7bit and quoted-printable in there.